doc/dev/release_checklist.md

```diff
@@ -124,6 +126,7 @@ We'll use `v4.6.0` as the intended release version as a running example.
       - Create and push the tag
       - Merge the tag into `stable`
 - Run `script/release_checklist.py v4.6.0` again to check that everything is in order.
+- Deploy the reference manual for the release to the specified URL, replacing the version that was deployed for the RC.
```

Comment:
I don't know how to do this step! What am I actually meant to do here?

Reply:
That was a bit of a TODO... I'll get it fleshed out, then return here. The plan is to have it be something like "push a tag that looks like this to such-and-such repo", but the deployment script isn't yet written.

Reply:
I've now got the manual set up so pushing the `v4.X.Y.Z` tag is sufficient to deploy it. I hope the new instructions are clear enough, please let me know if not.